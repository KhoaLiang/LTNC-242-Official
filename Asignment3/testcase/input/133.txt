INSERT x number
INSERT y string
INSERT z number
BEGIN
INSERT x number
INSERT y number
PRINT
END