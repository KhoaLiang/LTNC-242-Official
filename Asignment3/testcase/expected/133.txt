success
success
success
success
success
z//0 x//1 y//1